main/src/ca/uwaterloo/flix/language/phase/Typer.scala

```diff
@@ -20,6 +20,7 @@ import ca.uwaterloo.flix.language.ast.Ast.LabelledPrecedenceGraph
 import ca.uwaterloo.flix.language.ast._
```

Comment:
These changes effectively make the rest of the Typer dead code, correct?

Reply:
We agree that the new system is used here / now, right? I see you continue with the found subst.

Reply:
Yes, everything in the old `TypeInference.scala` file dies.